src/main/java/com/google/ehub/data/CommentDataManager.java

```diff
@@ -0,0 +1,38 @@
+package com.google.ehub.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.util.*;
+
+public class CommentDataManager{
+
+  public void addItemComment(String message, long timestamp){
+    Entity commentEntity = new Entity("ItemPageComments");
```

Comment:
i would change the name of the entity kind for comments into something like "comment". Make sure to store it in a static final variable like this:
```private static final COMMENT_KIND_KEY = "comment";```

Reply:
Noted.